python/cudf/cudf/_libxx/parquet.pyx

```diff
@@ -101,61 +93,48 @@ cpdef read_parquet(filepath_or_buffer, columns=None, row_group=None,
     cudf.io.parquet.to_parquet
     """
 
-    # Setup reader options
```

Comment:
Could we add the new row_group_count parameter to the python api ? (will be needed for improved dask perf when reading multiple rowgroups)

Reply:
@OlivierNV sure can! What is a sane default value? Not really familiar with row_group_count.

Reply:
I think -1 is the default (which is treated as 1 for rowgroup-based read or ignored otherwise iirc)

Reply:
ok have that added. Thanks for pointing it out.